include/mbgl/style/image.hpp

```diff
@@ -14,6 +14,12 @@ namespace style {
 using ImageStretch = std::pair<float, float>;
 using ImageStretches = std::vector<ImageStretch>;
 
+enum class TextFit : uint8_t {
```

Comment:
We follow the C++ Core Guidelines, which state

> [Specify the underlying type of an enumeration only when necessary](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-underlying)

Especially in this case I think it makes sense to just use the default.

Reply:
Thanks, actually I wrote it this way by referring to the existing code (like the enums in the file: src/mbgl/style/types.cpp), I thought this could save some space because the default type of the enum in C++ is `int`.

Reply:
There is something to be said for it, but since the space saving is minimal sticking to the default might be better.

I will leave it up to your judgement!

Reply:
yeah, you're right, this enum won't be used very frequently, so it won't save too much space, it looks like using default to keep the code clear is better.